docs/zh_cn/notes/faq.md

```diff
@@ -1,20 +1,68 @@
-# 常见问题解答（待更新）
+# 常见问题解答
 
 我们在这里列出了使用时的一些常见问题及其相应的解决方案。 如果您发现有一些问题被遗漏，请随时提 PR 丰富这个列表。 如果您无法在此获得帮助，请使用 [issue模板](https://github.com/open-mmlab/mmdetection/blob/master/.github/ISSUE_TEMPLATE/error-report.md/)创建问题，但是请在模板中填写所有必填信息，这有助于我们更快定位问题。
 
-## MMCV 安装相关
+## 安装
 
 - MMCV 与 MMDetection 的兼容问题: "ConvWS is already registered in conv layer"; "AssertionError: MMCV==xxx is used but incompatible. Please install mmcv>=xxx, \<=xxx."
 
-  请按 [安装说明](https://mmdetection.readthedocs.io/zh_CN/latest/get_started.html#installation) 为你的 MMDetection 安装正确版本的 MMCV 。
+  MMDetection，MMEngine 和 MMCV 的版本兼容关系如下。请选择合适的版本避免安装错误 。
+
+  | MMDetection 版本 |        MMCV 版本        |      MMEngine 版本       |
+  | :--------------: | :---------------------: | :----------------------: |
+  |       3.x        | mmcv>=2.0.0rc1, \<2.1.0 | mmengine>=0.5.0, \<0.7.0 |
+  |     3.0.0rc0     | mmcv>=2.0.0rc1, \<2.1.0 | mmengine>=0.5.0, \<0.7.0 |
+
+  **注意：**
+
+  1. 如果你希望安装 mmdet-v2.x, MMDetection 和 MMCV 版本兼容表可以在 [这里](https://mmdetection.readthedocs.io/en/stable/faq.html#installation) 找到，请选择合适的版本避免安装错误。
+  2. 在 MMCV-v2.x 中，`mmcv-full` 改名为 `mmcv`，如果你想安装不包含 CUDA 算子的版本，可以选择安装 MMCV 精简版 `mmcv-lite`。
 
 - "No module named 'mmcv.ops'"; "No module named 'mmcv.\_ext'".
 
-  原因是安装了 `mmcv` 而不是 `mmcv-full`。
+  原因是安装了 `mmcv-lite` 而不是 `mmcv`。
+
+  1. `pip uninstall mmcv-lite` 卸载安装的 `mmcv-lite`
```

Comment:
may also uninstall mmcv-full?

Reply:
I think this issue may check later, it is hard to check now

Reply:
Firstly, we cannot install mmcv-full>=2.0.rc0. Secondly, if we installed mmcv-full v1.x, the error may mmcv version is not match with mmdetv3.x.